run-tests.sh

```diff
@@ -13,36 +13,67 @@ set -o errexit
 # Quit on unbound symbols
 set -o nounset
 
+check_script () {
+    shellcheck run-tests.sh
+}
+
+check_black () {
+    black --check .
+}
+
+check_pydocstyle () {
+    pydocstyle cernopendata_client
+}
+
+check_flake8 () {
+    flake8 .
+}
+
+check_manifest () {
+    check-manifest
+}
+
+check_sphinx () {
+    sphinx-build -qnNW docs docs/_build/html
+    sphinx-build -qnNW -b doctest docs docs/_build/doctest
+}
+
+check_pytest () {
+    python setup.py test
+}
+
 for arg in "$@"
 do
     case $arg in
+        --check-shellscript) # Runs shell script static analysis
```

Comment:
These comments can go into functions (or, if transformed as `echo` statement, can even go away) so that here the `case` program can be nicely simplified... e.g. all in one line...

Reply:
@tiborsimko I have these comments as GA steps name in CI so I can remove the comments from shell script.